lib/rufus/lua/lib_lua_5_3.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Rufus::Lua
+  module Lib
+    module Lua5_3
+      def self.included(mod)
+        mod.attach_function :lua_callk, [ :pointer, :int, :int, :int, :int ], :void
+
+        mod.attach_function :lua_pcallk, [ :pointer, :int, :int, :int, :int, :int ], :int
+
+        mod.attach_function :lua_tonumberx, [ :pointer, :int, :pointer ], :double
+
+        mod.attach_function :lua_rawlen, [ :pointer, :int ], :int
+
+        mod.attach_function :lua_rotate, [ :pointer, :int, :int ], :void
+
+        mod.attach_function :luaL_loadbufferx, [ :pointer, :string, :int, :string, :pointer ], :int
+
+        mod.attach_function :lua_rawlen, [ :pointer, :int ], :int
+
+        mod.attach_function :lua_setglobal, [ :pointer, :string ], :void
+        mod.attach_function :lua_getglobal, [ :pointer, :string ], :int
+        mod.attach_function :luaL_requiref, [ :pointer, :string, :cfunction, :int ], :void
+
+        mod.extend(ClassMethods)
+      end
+
+      module ClassMethods
+        def lua_call(pointer, nargs, nresults)
```

Comment:
`lua_call` got moved to a C Macro, and isn't available publically anymore:
https://www.lua.org/source/5.3/lua.h.html#lua_call